docs/eventing/samples/ping-source/README.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "PingSource example"
+linkTitle: "PingSource"
+weight: 10
+type: "docs"
+---
+
+PingSource example shows how to configure PingSource as event source for
+functions.
+
+## Before you begin
+
+1. Setup [Knative Serving](../../../serving).
+1. Setup [Knative Eventing](../../../eventing).
+
+## Create a Knative Service
+
+In order to verify `PingSource` is working, we will create a simple Knative
+Service that dumps incoming messages to its log.
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display
+```
+
+Use following command to create the service from `service.yaml`:
+
+```shell
+kubectl apply --filename service.yaml
+```
+
+## Create a PingSource
+
+For each set of ping events you want to request, you need to create an Event
```

Comment:
"For each set of ping events you want to request" - would someone set up a bunch of different PingSource event sources though for each service, or would they have one PingSource and then use the broker and filters to send the events to different services? I don't think it's clear how this is used.